gcore/Cargo.toml

```diff
@@ -8,5 +8,9 @@ license = "GPL-3.0"
 [features]
 debug=[]
 
+[dependencies]
+gear-core-errors = { path = "../core-errors" }
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false }
```

Comment:
there should no be codec dependency. gcore is a minimal implementation. provide just first byte as the num of the error for it.

Reply:
I cannot, because error like `A::B(B::C(C::Example))` will be encoded like `[0, 0, 1]`

Reply:
ok, throw just bytes, let it be on the user's shoulders